packages/core/src/binary-data/object-store/s3.config.ts

```diff
@@ -23,20 +23,17 @@ class S3CredentialsConfig {
 }
 
 @Config
-class S3Config {
+export class S3Config {
 	/** Host of the n8n bucket in S3-compatible external storage @example "s3.us-east-1.amazonaws.com" */
 	@Env('N8N_EXTERNAL_STORAGE_S3_HOST')
 	host: string = '';
 
-	@Nested
-	bucket: S3BucketConfig;
+	@Env('N8N_EXTERNAL_STORAGE_S3_PROTOCOL')
+	protocol: 'http' | 'https' = 'https';
 
 	@Nested
-	credentials: S3CredentialsConfig;
-}
+	bucket: S3BucketConfig = new S3BucketConfig();
```

Comment:
Do we need these instantiations here? In other config classes nested configs are not instantiated